shared/wallets/transaction/index.js

```diff
@@ -383,9 +383,7 @@ export const Transaction = (props: Props) => {
             <Box2 direction="horizontal" fullWidth={true} style={styles.marginTopXTiny}>
               {props.onCancelPayment && (
                 <Box2 direction="vertical" gap="tiny">
-                  <Text type="BodySmall">
-                    {props.counterparty} can claim this when they set up their wallet.
-                  </Text>
+                  <Text type="BodySmall">{props.statusDetail}</Text>
```

Comment:
May want to pipe this through to [here](https://github.com/keybase/client/blob/master/shared/chat/conversation/messages/account-payment/container.js#L80) too